### ISSUE TITLE
Mobile: Implementing Static Profile Screen

### ISSUE THREAD

**Issue by bahricanyesil** [bounswe/bounswe2022group2#397]
### Issue Description

As we have talked about in our second meeting with the mobile team, I will implement a basic/static (not have a back connection) profile screen. The profile screen will include widgets for profile picture, email, username, and so on.

### Step Details

Steps that will be performed:
- [x] Integrate image picker package
- [x] Implement profile photo widget (selection and display)
- [x] Implement email text widget
- [x] Implement username text widget (editable)
- [x] Friends number text
- [x] Total enrolled course number
- [x] Total contributed course number
- [x] Integrate graph package for weekly activeness
- [x] Update button 

### Final Actions

I will test the profile screen on a local emulator. If everything goes well, I will create another branch on our repo and will be committing and pushing the changes into that branch. I will also share the development details under this issue. Finally, I will create a PR and after at least 1 person from the mobile team approved the pull request, it will be merged to the main branch.

### Deadline of the Issue

30.10.2022 - Sunday - 23:59

### Reviewer

Ezgi Aysel Batı

### Deadline for the Review

31.10.2022 - Monday - 23:59

**Comment by bahricanyesil** [comment]
I've completed all steps mentioned in the issue description. Now, the profile page is ready for the demo. I tried to put some dynamic value on the screen. I stored the email, username, and profile photo of the user in local storage and used them dynamically as the user updated.

I opened a [PR](https://github.com/bounswe/bounswe2022group2/pull/400) and now it is ready to be reviewed by the mobile team (@ezgy, @egemenatikk, @onurkomurcu)
